Source/Core/VideoBackends/Vulkan/VulkanContext.cpp

```diff
@@ -679,9 +744,8 @@ bool VulkanContext::CreateDevice(VkSurfaceKHR surface, bool enable_validation_la
   // Enable debug layer on debug builds
   if (enable_validation_layer)
   {
-    static const char* layer_names[] = {"VK_LAYER_LUNARG_standard_validation"};
     device_info.enabledLayerCount = 1;
-    device_info.ppEnabledLayerNames = layer_names;
+    device_info.ppEnabledLayerNames = &VALIDATION_LAYER_NAME;
```

Comment:
Confirming here that `VK_LAYER_LUNARG_standard_validation` was also deprecated for ` VK_LAYER_KHRONOS_validation` per [here](https://android.googlesource.com/platform/external/vulkan-validation-layers/+/HEAD/layers/README.md#standard-validation) and [here](https://www.lunarg.com/wp-content/uploads/2019/04/UberLayer_V3.pdf), at least as far as I can tell, so this change does make sense.